<launch>
  <node pkg="hengel_camera" type="virtualMap.py" name="hengel_camera_compensation"  output="screen" />
  <node pkg="hengel_camera" type="undistort.py" name="undistortion" />
</launch>
